src/main/java/com/facebook/openwifirrm/Constants.java

```diff
@@ -0,0 +1,13 @@
+package com.facebook.openwifirrm;
+
+public final class Constants {
```

Comment:
These constants are also used in tests that do not correspond to uCentral (e.g., in `ApiServerTest`). Should I still move it to ucentral? (I could go back to using String literals in the tests that don't test ucentral).

Also, do you mean "not good modular design" as in it's always kind of bad practice to have global constants or only bad when it could logically be moved lower but isn't?

Reply:
- You can import from "ucentral" in `ApiServer`, it is a dependency already.
- (IMO) global constants are usually bad in a non-trivial project since they can usually be split up logically.